<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuSystem</class>
 <widget class="QWidget" name="MenuSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuSystem</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>I am system setting window</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutoSearch">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Search</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnManualSearch">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Search</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSystemSetting">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>System Setting</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFactory">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Factory Set</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNetworkSetting">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Network Setting</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVersion">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCACard">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>CA Card</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
